cmake/partition_manager.cmake

```diff
@@ -158,9 +158,13 @@ if (CONFIG_PM_EXTERNAL_FLASH)
     )
 endif()
 
-set(pm_out_partition_files ${ZEPHYR_BINARY_DIR}/../partitions_${DOMAIN}.yml)
-set(pm_out_region_files ${ZEPHYR_BINARY_DIR}/../regions_${DOMAIN}.yml)
-set(pm_out_dotconf_files ${ZEPHYR_BINARY_DIR}/../pm_${DOMAIN}.config)
+if (DOMAIN)
+  set(domain_suffix _${DOMAIN})
+endif()
+
+set(pm_out_partition_files ${ZEPHYR_BINARY_DIR}/../partitions${domain_suffix}.yml)
+set(pm_out_region_files ${ZEPHYR_BINARY_DIR}/../regions${domain_suffix}.yml)
+set(pm_out_dotconf_files ${ZEPHYR_BINARY_DIR}/../pm${domain_suffix}.config)
```

Comment:
Should probably just be called `pm_out_<foo>_file`, as it is not a list of files.